tests/store/filters/test_status_cases_filters.py

```diff
@@ -623,3 +624,39 @@ def test_filter_case_by_empty_internal_id(store_with_multiple_cases_and_samples:
 
     # THEN the query should return no cases
     assert cases.count() == 0
+
+
+def test_get_active_cases_no_running_cases(store_with_multiple_cases_and_samples: Store):
+    # GIVEN a store containing cases with no "running" action
+    cases_query: Query = store_with_multiple_cases_and_samples._get_query(table=Family)
+    cases_query = cases_query.filter(Family.action != "running")
+
+    # WHEN getting active cases
+    active_cases: Query = get_active_cases(cases=cases_query)
+
+    # THEN the query should return no cases
+    assert active_cases.count() == 0
+
+
+def test_get_active_cases_with_running_cases(store_with_multiple_cases_and_samples: Store):
+    # GIVEN a store containing cases with at least one "running" action
+    cases_query: Query = store_with_multiple_cases_and_samples._get_query(table=Family)
```

Comment:
maybe not necessary but it would be good some assertion that verifies the assumption in the GIVEN statement